### ISSUE TITLE
Spart files - with and without allele information?

### ISSUE THREAD

**Issue by mvences** [iTaxoTools/Hapsolutely#2]
Here is one other comment by Aurelien:

"Among the set of exemple, one is a spart file (associated to the exemple 3).
begin spart;
Project_name = phased.tsv;
Date = 2023-08-19T12:00:00;
N_spartitions = 2 : three_species / two_species;
N_individuals = 14 / 14;
N_subsets = 3 / 2;
Individual_assignment =
specimen1a : 1 / 1
specimen1b : 1 / 1
specimen2a : 1 / 1
specimen2b : 1 / 1
specimen3a : 1 / 1
specimen3b : 1 / 1
specimen4a : 2 / 2
specimen4b : 2 / 2
specimen5a : 2 / 2
specimen5b : 2 / 2
specimen6a : 3 / 2
specimen6b : 3 / 2
specimen7a : 3 / 2
specimen7b : 3 / 2;
end;
This file works well, but it is apparently based on haplotypes, not on specimens (each specimen attribution to a subset is duplicated with a and b suffixes)."

So basically, what Aurelien suggests is that a Spart file should also work in attributing partitions if it includes the individual names without allele designation, in this case:
begin spart;
Project_name = phased.tsv;
Date = 2023-08-19T12:00:00;
N_spartitions = 2 : three_species / two_species;
N_individuals = 14 / 14;
N_subsets = 3 / 2;
Individual_assignment =
specimen1 : 1 / 1
specimen2 : 1 / 1
specimen3 : 1 / 1
specimen4 : 2 / 2
specimen5 : 2 / 2
specimen6 : 3 / 2
specimen7 : 3 / 2
end;
